docs/where-to-go-for-help.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: default
+title: Where To Go for Help
+---
+
+# Where To Go for Help
+
+Per the *FIRST* Role Description, the AV volunteer reports to the Event Coordinator (EC), but you can get help for some things from other people at the event or on Slack.
+
+For technical questions with the AV cart or any software on it, talk to someone in the **#av-help channel on Slack**.
```

Comment:
Make #av-help link to the channel
https://first-in-michigan.slack.com/archives/C04PTSR77H7